aiogram/api/client/session/base.py

```diff
@@ -12,30 +12,41 @@
 from ..telegram import PRODUCTION, TelegramAPIServer
 
 T = TypeVar("T")
-PT = TypeVar("PT")
+_JSON_LOADS = Callable[..., Any]
+_JSON_DUMPS = Callable[..., str]
```

Comment:
Why types named in uppercase?

Reply:
whoops caps. gotta edit JSON will remain uppercase tho. _JsonLoads should be fine, right?

Reply:
Yep